File: client/src/models/Sale.js
export class Sale {
    constructor({id, fullName, customerId, saleTotalItems, saleDate, billingDate, saleTotalValue, item}) {
        this.id = id;
        this.fullName = fullName;
        this.customerId = customerId;
        this.saleTotalItems = saleTotalItems;
        this.saleDate = saleDate;
        this.billingDate = billingDate;
        this.saleTotalValue = saleTotalValue;
        this.item = item;
    }
}

File: dashboard-lyncas/server/Dashboard.Domain/Models/Sale.cs
namespace Dashboard.Domain.Models
{
    public class Sale : BaseEntity
    {
        public DateTime SaleDate { get; set; }
        public DateTime BillingDate { get; set; }
        public double SaleTotalValue { get; set; }
        public double SaleTotalItems { get; set; }

        public int CustomerId { get; set; }
        public virtual Customer Customer { get; set; }

        public virtual List<ItemSale> SaleItems { get; set; }
    }
}
